src/test/regress/sql/create_drop_database_propagation.sql

```diff
@@ -0,0 +1,121 @@
+
+
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts3'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :worker_1_port
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts4'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :worker_2_port
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts5'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :master_port
+create user create_drop_db_test_user;
+
+set citus.enable_create_database_propagation=on;
+
+CREATE DATABASE mydatabase
+    WITH TEMPLATE = 'template0'
+            OWNER = create_drop_db_test_user
+            CONNECTION LIMIT = 10
+            ENCODING = 'UTF8'
+            LC_COLLATE = 'C'
+            LC_CTYPE = 'C'
+            TABLESPACE = create_drop_db_tablespace
+            ALLOW_CONNECTIONS = true
+            IS_TEMPLATE = false;
+
+SELECT pd.datname, pd.encoding,
+pd.datistemplate, pd.datallowconn, pd.datconnlimit,
+pd.datcollate , pd. datctype  ,  pd.datacl, rolname AS database_owner,
+pa.rolname AS database_owner, pt.spcname AS tablespace
```

Comment:
The rolname column is used twice as a result from this query (I guess by accident).
```suggestion
pd.datcollate , pd. datctype  ,  pd.datacl,
pa.rolname AS database_owner, pt.spcname AS tablespace
```